datadog-opentelemetry/examples/propagator/src/server.rs

```diff
@@ -18,7 +18,7 @@ use opentelemetry_http::{Bytes, HeaderExtractor, HeaderInjector};
 use opentelemetry_sdk::{
     error::OTelSdkResult,
     logs::{LogProcessor, SdkLogRecord, SdkLoggerProvider},
-    trace::{SdkTracerProvider, SpanProcessor},
+    trace::{SdkTracerProvider, SpanProcessor, TracerProviderBuilder},
```

Comment:
Not sure why I needed these changes but linter was unhappy without them. Lmk if I did something wrong here.